rules/Proj0453.md

```diff
@@ -0,0 +1,35 @@
+# Proj0453: Using Microsoft.NET.Test.Sdk implies a test project
+The inclusing of `Microsoft.NET.Test.Sdk` implies that the project is a test
+project. There is no reason to include the [SDK](https://github.com/microsoft/vstest)
```

Comment:
this statement is quite strong, since there are actually other reasons to include it, but those are not that common

Reply:
Like? I could not think of one. (And If that is the case, we should document them)

Reply:
I've adjusted the text.